momento-sdk/src/main/java/momento/sdk/PreviewStorageClient.java

```diff
@@ -0,0 +1,68 @@
+package momento.sdk;
+
+import java.util.concurrent.CompletableFuture;
+import momento.sdk.exceptions.NotFoundException;
+import momento.sdk.responses.storage.control.CreateStoreResponse;
+import momento.sdk.responses.storage.control.DeleteStoreResponse;
+import momento.sdk.responses.storage.control.ListStoresResponse;
+import momento.sdk.responses.storage.data.DeleteResponse;
+import momento.sdk.responses.storage.data.GetResponse;
+import momento.sdk.responses.storage.data.SetResponse;
+
+public class PreviewStorageClient implements IPreviewStorageClient, AutoCloseable {
+  /** Control operations */
+  public CompletableFuture<CreateStoreResponse> createStore(String storeName) {
+    return CompletableFuture.completedFuture(new CreateStoreResponse.Success());
+  }
+
+  public CompletableFuture<DeleteStoreResponse> deleteStore(String storeName) {
+    return CompletableFuture.completedFuture(new DeleteStoreResponse.Success());
+  }
+
+  public CompletableFuture<ListStoresResponse> listStores() {
+
+    return CompletableFuture.completedFuture(new ListStoresResponse.Success());
+  }
+
+  /** Data operations */
+  public CompletableFuture<GetResponse> get(String storeName, String key) {
+    if (key.equalsIgnoreCase("byte array")) {
```

Comment:
You can go ahead and wirte this stuff up to the protos, right?

Reply:
I added these mocks so the driver program was usable.

Reply:
that makes sense, just checking that the plan is to fill them in as later commits on this pr.

Reply:
I wrote the data and control clients but left this as is for test driving.